website/src/pages/users.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import Layout from '@theme/Layout';
+function Users() {
+  return (
+    <Layout title="Users">
+      <div
+        style={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          height: '50vh',
+          fontSize: '20px',
+        }}>
+        <p>
+          Edit <code>pages/hello.js</code> and save to reload.
```

Comment:
You need to migrate the v1 equivalent here. (just Reactjs knowledge).

Now its only “edit pages/hello.js”